src/request-api.js

```diff
@@ -173,33 +144,21 @@ exports = module.exports = function getRequestAPI (config) {
    *   buffer: // buffer the request before sending it - type: bool
    * }
    */
-  const send = function (options, callback) {
+  const send = (options, callback) => {
     if (typeof options !== 'object') {
       return callback(new Error('no options were passed'))
     }
 
     return requestAPI(config, options, callback)
   }
 
-  // Wraps the 'send' function such that an asynchronous
-  // transform may be applied to its result before
-  // passing it on to either its callback or promise.
```

Comment:
Added the description back and trying to describe as simple as I can what it does. What do you think?